src/Products/MeetingCommunes/adapters.py

```diff
@@ -354,27 +354,28 @@ def _insertItemInCategory(self, categoryList, item, byProposingGroup, groupPrefi
 
     security.declarePublic('getNumberOfItems')
 
-    def getNumberOfItems(self, itemUids, privacy='*', categories=[], list_types=['normal']):
+    def getNumberOfItems(self, itemUids, privacy=['public', 'secret'], categories=[], classifiers=[],
+                         list_types=['normal']):
         '''Returns the number of items depending on parameters.
            This is used in templates to know how many items of a particular kind exist and
            often used to determine the 'firstNumber' parameter of getPrintableItems/getPrintableItemsByCategory.'''
         # sometimes, some empty elements are inserted in itemUids, remove them...
-        itemUids = [itemUid for itemUid in itemUids if itemUid != '']
-        if not categories and privacy == '*':
-            return len(self.context.get_items(uids=itemUids, list_types=list_types))
-        # Either, we will have to filter (privacy, categories, late)
-        filteredItemUids = []
-        uid_catalog = getToolByName(self.context, 'uid_catalog')
-        for itemUid in itemUids:
-            obj = uid_catalog(UID=itemUid)[0].getObject()
-            if not (privacy == '*' or obj.getPrivacy() == privacy):
-                continue
-            elif not (categories == [] or obj.getCategory() in categories):
-                continue
-            elif not obj.isLate() == bool(list_types == ['late']):
-                continue
-            filteredItemUids.append(itemUid)
-        return len(filteredItemUids)
+        if privacy == '*':
```

Comment:
du coup `if privacy and privacy != "*"` alors tu ajoutes "privacy" dans la query

Reply:
je ne pense pas non plus donc je l'ai viré c'est plus backward compat

Reply:
je crois qu'il vaut mieu que ça casse en cas de mauvais param. je me tate de mettre un raise ValueError si on lui donne une str